src/EmployerPR/SFA.DAS.EmployerPR.Api/Controllers/PermissionsController.cs

```diff
@@ -19,4 +20,15 @@ public async Task<IActionResult> GetPermissions([FromQuery] GetPermissionsQuery
 
         return Ok(result);
     }
+
+    [HttpPost]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(PostPermissionsCommandResult), StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
```

Comment:
Bad request is not being produced. Usually all the validations are happening in the FE, so these requests are not expected to fail validations in the inner, so we do not care about bad request, if it fails any validation in the inner then we just let the error bubble up.